chia/_tests/blockchain/test_blockchain.py

```diff
@@ -1018,17 +1045,13 @@ async def test_reset_deficit(self, empty_blockchain, bt):
             await _validate_and_add_block(empty_blockchain, blocks[-1])
 
     @pytest.mark.anyio
-    async def test_genesis_has_ses(self, empty_blockchain, bt):
+    async def test_genesis_has_ses(self, empty_blockchain: Blockchain, bt: BlockTools) -> None:
         # 3a
         block = bt.get_consecutive_blocks(1, skip_slots=1)[0]
         new_finished_ss = recursive_replace(
             block.finished_sub_slots[0],
             "challenge_chain",
-            recursive_replace(
-                block.finished_sub_slots[0].challenge_chain,
-                "subepoch_summary_hash",
-                bytes([0] * 32),
-            ),
+            recursive_replace(block.finished_sub_slots[0].challenge_chain, "subepoch_summary_hash", bytes32([0] * 32)),
```

Comment:
I don't think these formatting change belongs in this PR, nor do I think it's an improvement